test/src/test/java/hudson/model/QueueTest.java

```diff
@@ -1367,19 +1367,6 @@ public void regularTooltipDisplayedCorrectly() throws Exception {
         assertThat(tooltip, containsString(expectedLabel.substring(1, expectedLabel.length() - 1)));
     }
 
-    @Test
-    @Issue("SECURITY-1537")
-    public void preventXssInCauseOfBlocking() throws Exception {
```

Comment:
does this still need removing or could it be adapted?

Reply:
Looks like this is the only failing test left - doesn't want to seem to run locally and it's giving this on CI:

![image](https://user-images.githubusercontent.com/43062514/193406545-49f4fb51-83a7-4c4e-b38d-ea6a56e10b94.png)

Any ideas?